packages/cadl-rlc-emitter/src/transform/transformPaths.ts

```diff
@@ -19,18 +22,35 @@ export function transformPaths(program: Program): Paths {
   const [routes, _diagnostics] = getAllRoutes(program);
   const paths: Paths = {};
   for (const route of routes) {
+    const parameterBaseName = getParameterBaseName(
+      route.groupName,
+      route.operation.name
+    );
+    const respNames = [];
+    for (const resp of route.responses) {
+      const respBaseName = getResponseBaseName(
+        route.groupName,
+        route.operation.name,
+        resp.statusCode === "*" ? "Default" : resp.statusCode
+      );
+      const respName = getResponseTypeName(respBaseName);
```

Comment:
If you don't reuse the `respBaseName` you can call this way:
```suggestion
      const respName = getResponseTypeName(
       route.groupName,
        route.operation.name,
        resp.statusCode === "*" ? "Default" : resp.statusCode);
```